src/commands/synthetics/interfaces.ts

```diff
@@ -542,6 +542,7 @@ export interface SyntheticsCIConfig extends APIHelperConfig {}
 export interface RunTestsCommandConfig extends SyntheticsCIConfig {
   batchTimeout?: number
   configPath: string
+  buildCommand?: string
```

Comment:
I hate saying that, but can you alphabetically order it?

Reply:
Oh, good catch, thanks for raising that up 🙏